tests/e2e/utils/StringUtil.ts

```diff
@@ -87,4 +87,41 @@ export class StringUtil {
 		const regex: RegExp = new RegExp(`(${name})=[^&]+`, 'g');
 		return queryString.replace(regex, `$1=${replaceStr}`);
 	}
+
+	/**
+	 * compares two semantic version strings
+	 * @param version1 First version to compare (e.g., '4.18')
+	 * @param version2 Second version to compare (e.g., '4.18')
+	 * @returns -1 if version1 < version2, 0 if version1 = version2, 1 if version1 > version2
+	 */
+	static compareVersions(version1: string, version2: string): number {
```

Comment:
@martinszuc : what do you think about simplifying the code by reusing an existing npm library [compare-versions](https://www.npmjs.com/package/compare-versions)?